docs/reference/import.md

```diff
@@ -31,8 +32,8 @@ Unfortunately, copying data is not always feasible for the following reasons:
 To solve this we offer an import tool that does not copy any data, allowing for a more gradual onboarding process.
 
 The lakeFS import tool will use the [S3 Inventory](https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-inventory.html) feature to create lakeFS metadata.
-The imported metadata will be reflected in lakeFS under a special branch, called `import-from-inventory`.
-You should not make any changes or commit anything to this branch: it will be operated on only by lakeFS.
+The imported metadata will be reflected in lakeFS under a special branch, called `import-from-inventory`, unless it's the first commit to the repository that will be committed directly to `master`.
```

Comment:
```suggestion
In case the repository is empty, the imported metadata will be committed directly to the main branch. In all other cases, it will be committed to a special branch, called `import-from-inventory`.
```

Reply:
default branch